shed/event_streams.py

```diff
@@ -464,7 +464,16 @@ def issue_event(self, outputs):
                                        for (output_name, desc), output in
                                        zzip(self.output_info, outputs)})
             else:
-                new_event.update(data=outputs['data'])
+                if outputs is None:
+                    outputs = {}
+                elif not isinstance(outputs, dict):
+                    errormsg = "Error, outputs is not a dict. Can't continue"
```

Comment:
You might consider writing one big string:
```python
errormsg = '''Error, outputs is not a dict. Can't continue \n 
This typically comes from a function whose output is not nothing or a dict. 
When dealing with such outputs, please use the output_info keyword argument.'''
```
With the triple quote you can have as many line breaks as you like.

Reply:
I think I found a solution to this (of course after I merged, meh) https://docs.python.org/3.1/library/textwrap.html#textwrap.dedent